src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -137,6 +137,7 @@ const DropdownMenu = <T,>({
 
     return (
         <DropdownMenuContext.Provider value={contextValue}>
+            {/*as this div has Button component as child, clicking on it one fires onClick of this div on bubbling*/}
```

Comment:
i guess we should make breaking change in future and always add `onClick` to toggle control

Reply:
if button has onClick handler also, it may lead to double-handling of click event. Should be accurate there